frontend/integration-tests/tests/event.scenario.ts

```diff
@@ -0,0 +1,58 @@
+import { browser, $, element, by, ExpectedConditions as until } from 'protractor';
+import { appHost, checkLogs, checkErrors, testName } from '../protractor.conf';
+import { safeLoad, safeDump } from 'js-yaml';
+import * as _ from 'lodash';
+import * as crudView from '../views/crud.view';
+import * as yamlView from '../views/yaml.view';
+
+describe('Events', () => {
+  const leakedResources = new Set<string>();
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  describe('Events', () => {
+    const name = `${testName}-pod`;
+    const testLabel = 'automatedTestName';
+
+    it('displays Events all resources', async () => {
+      await browser.get(`${appHost}/k8s/all-namespaces/events`);
+      await browser.wait(until.presenceOf($('.co-sysevent__box')));
+      expect($('.co-sysevent__box').isPresent()).toBe(true);
+    });
+
+    it('creates a new pod resource instance for event testing', async () => {
+      await browser.get(`${appHost}/k8s/ns/${testName}/pods`);
+      await crudView.isLoaded();
+      await crudView.createYAMLButton.click();
```

Comment:
Why is the `creatYAMLButton.click()` needed?

Reply:
That's what takes me into the yaml editor, which I use to create the pod.